.devcontainer/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM rust:1.50
+
+# versions (according to their git tag) of different tools to download
+# https://github.com/EmbarkStudios/cargo-deny/releases
+ARG CARGO_DENY_VER=0.9.0
+# https://github.com/rust-lang/mdBook/releases
+ARG MDBOOK_VER=v0.4.7
+# https://github.com/rustwasm/wasm-pack/releases
+ARG WASM_PACK_VER=v0.8.1
+# https://nodejs.org/
+ARG NODE_VER=v14.16.0
```

Comment:
I'd like to stick with node 12.x for for now (currently 12.20.1) to maintain compatibility with our systems that use the web assemblies. That way our node tests use the same version of node for testing that the _users_ do.

Reply:
I updated the Dockerfile to take a `NODE_MAJOR_VERSION` and switched it to `12`. It now has some more complex logic to go and get the latest version for whatever major version specified.